src/main.cpp

```diff
@@ -158,10 +158,16 @@
 #define SerialPort Serial
 
 #if defined(LED_POWER_PIN)
+	#if !defined(LED_POWER_INVERT)
+		#define LED_POWER_INVERT false
```

Comment:
This is not necessary. use #ifdef LED_POWER_INVERT or #if defined(LED_POWER_INVERT) in the code later. I guess the only place where it may be neccesery is for currentPowerPinMode if you need to switch the current state.

Reply:
Yes, this can be removed, as  the `#if POWER_LED_INVERT` statement in powercontrol.h will set the inverted methods if POWER_LED_INVERT is set to true, and set them non-inverted if set to false or not defined.